edap.py

```diff
@@ -232,8 +228,15 @@ def update_parser(parser=None):
     return parser
 
 
+class Edap(LdapObjectsMixin, LdapGroupMixin, LdapUserMixin, LdapFranchiseMixin, LdapDivisionMixin):
+
+    def __init__(self, bound_ldap):
```

Comment:
I would be in favor of transfering `BoundLdap` to `Edap`, so one would write

```
edap = Edap(args.hostname, args.admin_dn, args.password)
```

instead of
```
ld = BoundLdap(args.hostname, args.admin_dn, args.password)
edap = Edap(bound_ldap=ld)
```

Reply:
Done